chronicler/main.go

```diff
@@ -1,7 +1,76 @@
 package main
 
+import (
+	"context"
+	"encoding/json"
+	"log/slog"
+	"os"
+	"time"
+
+	"github.com/LeperGnome/simple-chat/pkg/chronicler"
+	"github.com/segmentio/kafka-go"
+)
+
+const (
+	topicName = "messages"
+	kafkaAddr = "kafka:9092"
+)
+
+type Message struct {
+	From    string `json:"from"`
+	Content string `json:"content"`
+	Channel string `json:"channel"`
+}
+
 func main() {
-	// TODO:
-	// read kafka messages topic
-	// write it to db
+	repo, err := chronicler.NewRepository(os.Getenv("DB_USER"), os.Getenv("DB_PASSWORD"), os.Getenv("DB_HOST"), os.Getenv("DB_NAME"))
```

Comment:
Can use envconfig

Reply:
Added, but don't know where to put it, to reuse between two services. Packages feels like a wrong place to have any deps related to env vars. Maybe it's okay to just duplicate it. idk